### ISSUE TITLE
PHP 8.0 Imagick not supported / causes error

### ISSUE THREAD

**Issue by simonkey** [lando/platformsh#138]
The extension imagick cannot be enabled for the platform.sh recipe when using php 8.0. There was an issue/bug in the platform.sh image which has been resolved.

Setup direct in `.platform.app.yaml` or via override.

```yaml
type: php:8.0

runtime:
  extensions:
     - imagick
     - ...
``` 
Causes the following error: 
```
Exception: Extension '{'configuration': {}, 'name': u'imagick'}' is unknown. 
```

The deployment to platform.sh works without issues, the error only occurs locally. If I override the extensions and remove `imagick`, then it's possible to start the lando project. Maybe there is the latest image for php 8.0 missing / wrong tagged.

**Comment by valentin-** [comment]
Any progress on this issue?

**Comment by sgelliott** [comment]
@simonkey I think this issue is probably related to this one: #135 

Are you only seeing this with php:8.x or also for 7.4?

**Comment by simonkey** [comment]
@sgelliott Yes, it only happens with` type: "php:8.0"`. But redis is working without problemes. My machine is running Ubuntu 20.04 with an Intel Processor. So the redis issue could be only Apple Silicon related.

**Comment by sgelliott** [comment]
@simonkey I get the redis issue with php:8.1, not with 8.0, and imagick with any 8.x, (all on Intel Mac) but that's being tracked in the other issue. So seems like issues with extensions are centering around different php versions & different configurations.

**Comment by simonkey** [comment]
@sgelliott I've tried different php extensions and as you stated `imagick` doesn't work in 8.x versions. And I can confirm that `redis` isn't working for `php:8.1`. There is another extension which isn't working in `php:8.1` and that is `igbinary`.

**Comment by simonkey** [comment]
Any updates on this issue?

**Comment by gilzow** [comment]
Just ran into this same issue with the `imagik` extension and `type: "php:8.1"` 
Any updates?
